custom_components/ariston/manifest.json

```diff
@@ -9,7 +9,7 @@
   "iot_class": "cloud_polling",
   "issue_tracker": "https://github.com/fustom/ariston-remotethermo-home-assistant-v3/issues",
   "loggers": ["ariston"],
-  "requirements": ["ariston==0.12.3"],
+  "requirements": ["git+https://github.com/gamba69/python-ariston-api@main#ariston==1.0.0"],
```

Comment:
This change is useless on main repo.

Reply:
I don't know what version of api will be at the moment of pull request merging. This change help me install intergation from fork repository and test it. In this case HA download api from api forked repo...